src/badge.py

```diff
@@ -446,7 +446,8 @@ def sync_timestamp(self):
                     self.sendIdentifyReq(10)
 
                 if self.dlg.timestamp_sec != 0:
-                    self.logger.info("Badge datetime was: {},{} and Badge id : {} , and project id : {}".format(self.dlg.timestamp_sec, self.dlg.timestamp_ms,self.badge_id,self.project_id))
+                    self.logger.info("Badge datetime was: {},{}".format(self.dlg.timestamp_sec, self.dlg.timestamp_ms))
+                    self.logger.info("Badge id : {} , and project id : {}".format(self.badge_id, self.project_id))
```

Comment:
Better. I know i'm being picky, but please move this line before line 441 (just before sending the status request), since we want to show this message very time we call this function, so whoever reads the logs can see that we are sending the IDs.

(My original code is very, very messy. I know it is difficult to see this)